libraries/plugins/tags/include/scorum/tags/tags_api_objects.hpp

```diff
@@ -177,10 +177,24 @@ struct discussion_query
     /// query limit
     uint32_t limit = 0;
 
-    bool tags_logical_and = true;
+    /// require that all tags in query must exist in querying posts
+    bool all_tags_exist = true;
```

Comment:
if the reason for this rename is a better name than please leave original to avoid code changes on frontend.